### DISCUSSION TITLE
Bacnet stack 1.0

### DISCUSSION THREAD

**Discussion by sangeetamutgi** [bacnet-stack/bacnet-stack#401] [Q&A]
I have couple of questions before we upgrade from bacnet-stack-0.8.6
1)  Is the bacnet-stack-1.0 tag branch stable to use ?
2) I get following 2 errors when we run the Bacnet Certification test on bacnet-stack-0.8.6 . Are these 2 fixed in the bacnet-stack-1.0
a)  ReadRange a non-Array List Property with an Array Index - 
                 Got Error Code=Property is not a List - 
                 expected : Property_Is_Not_an_Array
b) When an unknown network layer message is received 
                got timeout error ;
               expected =  REJECT_MESSAGE_TO_NETWORK 

Any response will be helpful
Thanks  Sangeeta

**Comment by skarg** [comment]
Most of the releases are stable.  We use dozens of builds and quality tests in the pipeline to help ensure that things don't get broken. However, there is always a chance that something was missed.  

Please post each failing tests as a separate github issue with any information about recreating the problem. There is an example BACnet device server application (bacserv) that can be run on most OS platforms (Mac/BSD, Linux, Windows) that includes most of BACnet Stack objects and services and could be the target of a BACnet certification test for a server device.
